default/generated/spring-framework/tests/data/data-access/jpa-data-access/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Lets use the existing project in `data/data-access` - basically move this project up one step and merge the poms. We just need to make sure that all the tests for data-access still pass.